.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+Contributing to Goss
+====================
+
+Thank you for your interest in contributing to Goss. Goss wouldn't be where it is today if it wasn't for people like you. Some ways you can contribute:
+
+* Improve the [README](https://github.com/aelsabbahy/goss/blob/master/README.md) and/or [Docs](https://github.com/aelsabbahy/goss/blob/master/docs/manual.md) - This makes it easier for new users to learn goss.
+* Vote on bugs and feature requests by adding a thrumbs up to the inital post.
+* Create tutorials, blog posts and example use-cases on how to use Goss.
+* Help users with [questions](https://github.com/aelsabbahy/goss/labels/question) tracker.
+* Fix verified [bugs](https://github.com/aelsabbahy/goss/labels/bug).
+* Implement approved [feature requests](https://github.com/aelsabbahy/goss/labels/feature).
+* Spread the word.
+
+
+Features and bug reports and questions
+--------------------------------------
+
+Please search the [issues](https://github.com/aelsabbahy/goss/issues) page before opening a feature request or a bug report. If a feature or a bug report already exists, please thumbs up the initial post to indicate it's importance to you and raise it's priority. Please comment and contribute to said issue if you feel it's deficient.
+
+Bug reports
+-----------
+
+If you think you found a bug in Goss, please submit a [bug report](https://github.com/aelsabbahy/goss/issues).
+
+Feature requests
+----------------
+
+If there's a feature you wish Goss would support, please open a feature request.
+
+Some things to note prior to opening a Goss feature request:
+* Goss is intended to be quick and easy to learn.
+* Goss is focused on the 20% of the 80/20 rule. In other words, Goss focuses on the 20% of features that cover the core aspects of OS testing and benefit 80% of users.
+* Goss provides a generic [command](https://github.com/aelsabbahy/goss/blob/master/docs/manual.md#command) runner to allow users to cover more nuanced test cases.
+
+If you believe your feature adheres to the goals of Goss, please open a [feature request](https://github.com/aelsabbahy/goss/issues) on GitHub which describes the feature you would like to see, why it is useful, and how it should work.
```

Comment:
```suggestion
If you believe your feature adheres to the goals of Goss, please open a [feature request](https://github.com/aelsabbahy/goss/issues) on GitHub which describes the problem you would like solved, why it is useful, and how it should work.
```

Maybe this is my semantics, but this encourages I think people to think in terms of "I need x to accomplish y" vs "It would be cool if...".